### PR TITLE
expanded sections

### PR DESCRIPTION
**Pull request by masterfung**
- transaction docs expanded with EIP-712, txTypes, and overview
- updated a few dependencies
- added more areas for visibility on Testnet (Keplr and Metamask)
- better handling of errors with sweetalerts

### CODE REVIEW

File: docs/develop/index.mdx [hunk 1]
```diff
@@ -4,6 +4,18 @@ sidebar_position: 1
 
 # Overview
 
+
```
Comment (danburck) [inline docs/develop/index.mdx:7]: Move this to the Testnet section

File: docs/protocol/concepts/transactions.md [hunk 2]
```diff
@@ -134,6 +134,64 @@ The reason of this, is because the `MsgEthereumTx` must not be included in a `au
...
+changes to Ethereum's fee market, with the aim of making gas fees more predictable and improving user experience.
+EIP-1559 transactions include two main components: a base fee and a priority fee (or tip). The base fee is algorithmically
+determined by the network, while the priority fee is set by users to incentivize miners to include their transaction.
+The base fee is burned, effectively reducing the overall ETH supply, while the priority fee goes to miners as a reward
+for their work. DynamicFeeTxType transactions allow for more predictable and efficient gas fee management.
+
+These transaction types represent Ethereum's continuous evolution and improvements to its network, helping address
+challenges related to scalability, security, and user experience.
+
+### EIP-712
```
Comment (danburck) [inline docs/protocol/concepts/transactions.md:166]: This isn't a transaction but rather a signing method. Would add to a new page called signing

File: docs/use/connect-your-wallet/keplr.mdx [hunk 3]
```diff
@@ -6,6 +6,17 @@ sidebar_position: 2
 
 Connect your Keplr wallet with Evmos.
 
+import { chainConfig, handler } from '@site/src/utils/ChainConfig';
```
Comment (danburck) [inline docs/use/connect-your-wallet/keplr.mdx:9]: would remove this here, as new users will want to connect to **Mainnet**

Comment (masterfung) [reply docs/use/connect-your-wallet/keplr.mdx:9]: Not sure I agree but I moved it to the testnet of this wallet section

**Review by danburck — APPROVED** [review]
LGTM

**Pull request merged**